loafer/ext/aws/providers.py

```diff
@@ -38,7 +38,7 @@ def __str__(self):
             queue_url = await self.get_queue_url(self.queue_name)
             response = await self.client.receive_message(QueueUrl=queue_url, **self._options)
         except (botocore.exceptions.BotoCoreError, botocore.exceptions.ClientError) as exc:
-            raise ProviderError('error fetching messages: {}'.format(str(exc))) from exc
+            raise ProviderError('error fetching messages from queue={}: {}'.format(str(exc), self.queue_name)) from exc
```

Comment:
@wiliamsouza I think the .format() params needs to be swapped ? (queue_name and then exc)